doc/classes/ProjectSettings.xml

```diff
@@ -343,7 +343,7 @@
 			This setting can be overridden using the [code]--frame-delay &lt;ms;&gt;[/code] command line argument.
 		</member>
 		<member name="application/run/low_processor_mode" type="bool" setter="" getter="" default="false">
-			If [code]true[/code], enables low-processor usage mode. This setting only works on desktop platforms. The screen is not redrawn if nothing changes visually. This is meant for writing applications and editors, but is pretty useless (and can hurt performance) in most games.
+			If [code]true[/code], enables low-processor usage mode. The engine optimizes for low processor usage by only refreshing the screen if needed and can improve battery consumption on mobile. This is meant for writing applications and editors, but is pretty useless (and can hurt performance) in most games.
```

Comment:
If I may chime in with a huge proposition:
```suggestion
			If [code]true[/code], enables low-processor usage mode. When enabled, the engine takes longer to redraw, but only redraws the screen if necessary. This may lower power consumption, and is intended for editors or mobile applications. For most games, because the screen needs to be redrawn every frame, it is recommended to keep this setting disabled.
```
"Improve battery consumption" can be interpreted as making the battery consume better, therefore more?

Reply:
Thanks!